lineapy/plugins/base.py

```diff
@@ -127,7 +127,7 @@ def slice_dag_helper(
         for slice_name in slice_names:
             artifact_var = safe_var_name(slice_name)
             slice_code: CodeSlice = get_program_slice_by_artifact_name(
-                self.db, slice_name
+                self.db, slice_name, keep_lineapy_save=True
```

Comment:
@mingjerli This is inside the method that creates a DAG from the sliced code. When building pipelines, we want to retain `lineapy.save()` in code slices because, otherwise, pipeline nodes simply perform computation without writing out their job results (that should be picked up by other nodes).

Reply:
If Ming-jer is confused it's likely that our users will be too.

Reply:
Okay, I can then add a clarifying in-line comment for this.